erpnext/manufacturing/doctype/work_order/work_order.json

```diff
@@ -13,6 +13,7 @@
   "item_name",
   "image",
   "bom_no",
+  "title",
```

Comment:
Below is the Image attached for Work Order custom form fields and Doctype field difference, where "Title" was the custom field which added to Work Order doctype fields.

https://user-images.githubusercontent.com/80836439/122998481-ba74bf80-d3ca-11eb-9d81-12e0a73d0470.mp4

![Work_order_before](https://user-images.githubusercontent.com/80836439/122998679-f871e380-d3ca-11eb-8a42-dcd1bff6d5ae.PNG)
![Work_order_before1](https://user-images.githubusercontent.com/80836439/122998691-fc056a80-d3ca-11eb-9710-84e2275d0fe8.PNG)